packages/expo-updates/ios/EXUpdates/AppController.swift

```diff
@@ -463,10 +463,10 @@ public class AppController: NSObject, AppLoaderTaskDelegate, ErrorRecoveryDelega
         updateId: update.loggingId(),
         assetId: nil
       )
-      stateMachine?.processEvent(UpdatesStateEventDownloadCompleteWithUpdate(manifest: update.manifest!.rawManifestJSON()))
+      stateMachine?.processEvent(UpdatesStateEventDownloadCompleteWithUpdate(manifest: update.manifest?.rawManifestJSON()))
       // Send UpdateEvents to JS
       sendLegacyUpdateEventToBridge(AppController.UpdateAvailableEventName, body: [
-        "manifest": update.manifest!.rawManifestJSON()
+        "manifest": update.manifest?.rawManifestJSON()
```

Comment:
did we ever send nil for the legacy update event manifest? It feels incorrect to me to send an `UpdateAvailableEventName` but nil for manifest.

Reply:
I think maybe we just want to not send the legacy event if there's no manifest (for SDK 49 only since)? For the state machine event either way works since there's fewer apps relying upon it.

Reply:
This seems to only affect the rollback test, and yes rollbacks should be broken on SDK 49 -- but I'm not sure that it's good to let a rollback directive crash the app. This seems to prevent the crash without breaking anything else.

Reply:
I've adjusted this to only send events if the manifest is non-null. Since the state machine is not used in SDK 49, and rollback is the only flow where this could happen, we can make this change to remove the force unwrapping that caused the CI failure.